dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -1,7 +1,9 @@
 /*
- * FILE:       dll/win32/rasdlg/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Ștefan Fulea (stefan dot fulea at mail dot com)
+ * PROJECT:     ReactOS RASDLG
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
```

Comment:
added PURPOSE line useless in all your PRs.

Reply:
Ok, I did it!

Reply:
Again, this was wrong suggestion. Please revert back.

Reply:
I understand, I will.